### ISSUE TITLE
Plotting issues with parameter scan

### ISSUE THREAD

**Issue by kirichoi** [sys-bio/tellurium#335]
In [parameter scan module](https://github.com/sys-bio/tellurium/blob/master/tellurium/analysis/parameterscan.py) you can see that all plotting functions lacks plt.plot() which break things. 
I wonder why the module tries to save images and read it by default? 
```
        FILENAME = str(uuid.uuid4())+".png"
        plt.savefig(FILENAME)
        plt.close()
        imag = mpimg.imread(FILENAME)
        return(imag)
```

**Comment by kirichoi** [comment]
Solve this by adding something like:
```
if any('SPYDER' in name for name in os.environ):
    plt.plot()
else:
    FILENAME = str(uuid.uuid4())+".png"
    plt.savefig(FILENAME)
    plt.close()
    imag = mpimg.imread(FILENAME)
    return(imag)
```

**Comment by kirichoi** [comment]
Fixed by commit 9c6fe4ae7c3c5e5dc11063bafec1f727e86184ef
